docs/source/transaction_family_specifications/sawtooth_burrow_evm_family.rst

```diff
@@ -0,0 +1,269 @@
+stored under the prefix formed by taking the first 3 bytes of the SHA512 hash
+of the UTF-8 encoded string “burrow-evm”. This shall be referred to as the EVM
+prefix.::
+
+    >>> hashlib.sha512('burrow-evm'.encode('utf-8')).hexdigest()[0:6]
+    'a84eda'
+
+Account Addresses
+-----------------
+The address of an account depends on which type of account it is:
+
+* EOA - Given an EOA’s private key, the address is the, “rightmost 160-bits of
+  the Keccak hash of the corresponding ECDSA public key,” from the yellow paper.
+  The “Keccak hash” shall be taken to mean SHA3.
+* CA - The address of a CA is the last 20 bytes of the 256-byte SHA3 hash of
+  the byte array formed by concatenating the sender address and the big endian
+  encoding of the sender’s current nonce. In other words, the address of a CA is
+  derived from hashing the creating accounts address and its nonce.
+
+Accounts in the above format shall be stored in global state at the address
+formed by concatenating the EVM prefix, address of the account, and enough 0’s
+to form a valid global state address.::
+
+    >>> state_address = 'a84eda' + account_address + '0'*12
+
+Transaction Payload
+===================
+
+The transaction payload closely follows the structure as defined by the EVM
+yellow paper, section 4.3 (“The Transaction”). In the Sawtooth Burrow-EVM
+Transaction Family, the transaction payload shall be represented using the
+following protobuf message::
+
+    message EvmTransaction {
+        // "...the number of transactions sent by the sender; formally T_n."
+        uint64 nonce = 1;
+
+        // "...the number of Wei to be paid per unit of gas for all computation
+        // costs incurred as a result of the execution of this transaction;
+        // formally T_p."
+        uint64 gas_price = 2;
+
+        // "...the maximum amount of gas that should be used in executing this
+        // transaction. This is paid up-front, before any computation is done and
+        // may not be increased later; formally T_g"
+        uint64 gas_limit = 3;
+
+        // "...the 160-bit address of the message call's recipient or, for a
+        // contract creation transaction, {}, used here to denote (the empty byte
+        // array); formally T_t."
+        bytes to = 4;
+
+        // "...the number of Wei to be transferred to the message call's recipient
+        // or, in the case of contract creation, as an endowment to the newly
+        // created account; formally T_v."
+        uint64 value = 5;
+
+        // The yellow paper also includes a signature in the transaction, but this
+        // is already included as part of the Sawtooth transaction so we don't
+        // duplicate it here.
+
+        // "An unlimited size byte array specifying the EVM-code for the account
+        // initialisation procedure, formally T_i."
+        //
+        // This should only be set if this is a contract creation transaction.
+        bytes init = 7;
```

Comment:
I agree that would be a good idea, but it isn't relevant to this PR which is to document the decisions​ that have already been made.